### PR TITLE
Issue 283/replace deprecated activity test rule with ActivityScenarioTestRule

### PR DESCRIPTION
**Pull request by sergio-sastre**

### CODE REVIEW

**Review by RuslanMingaliev — DISMISSED** [review]
LGTM

**Review by matzuk — APPROVED** [review]

File: samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/compose_tests/WebComposeTest.kt [hunk 1]
```diff
@@ -20,98 +20,92 @@ class WebComposeTest : TestCase() {
...
-                        }
-
-                        compose {
-                            orWithElement(
-                                Locator.XPATH,
-                                "/html/body/div[1]/footer/div/div/div[1]/nav/div[1]/a"
-                            ) {
-                                hasText("TRATATATA")
+                            or {
+                                containsText("Ask questiop")
```
Comment (Nikitae57) [inline samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/compose_tests/WebComposeTest.kt:71]: nit: "questiop"?

File: samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/docloc_tests/advanced/ProductDocLocScreenshotTestCase.kt [hunk 2]
```diff
@@ -1,15 +0,0 @@
-package com.kaspersky.kaspressample.docloc_tests.advanced
-
-import androidx.test.rule.ActivityTestRule
-import com.kaspersky.kaspressample.docloc.FragmentTestActivity
-import com.kaspersky.kaspresso.testcases.api.testcase.DocLocScreenshotTestCase
-import org.junit.Rule
-
-open class ProductDocLocScreenshotTestCase : DocLocScreenshotTestCase(locales = "en,ru") {
```
Comment (Nikitae57) [inline samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/docloc_tests/advanced/ProductDocLocScreenshotTestCase.kt:8]: It's still mentioned in 07_DocLoc.md

**Review by Nikitae57 — CHANGES_REQUESTED** [review]
As a couple of types were removed, we need to update docs. nit: readme and *.md files under wiki folder still contain code with ActivityScenarioTestRule

**Comment by sergio-sastre** [comment]
> As a couple of types were removed, we need to update docs. nit: readme and *.md files under wiki folder still contain code with ActivityScenarioTestRule

Thanks for the hint!
I made the corresponding changes

**Review by eakurnikov — APPROVED** [review]

**Comment by sergio-sastre** [comment]
> > As a couple of types were removed, we need to update docs. nit: readme and *.md files under wiki folder still contain code with ActivityScenarioTestRule
> 
> Thanks for the hint! I made the corresponding changes

@Nikitae57 Could you review again, please?

**Comment by matzuk** [comment]
Hey @sergio-sastre!
Could you merge the latest master? We updated/fixed our examples.
@AzamatCherchesov please review too and merge the PR if everything is good.

**Comment by Nikitae57** [comment]
@sergio-sastre could you resolve conflicts and make sure that tests are green, please?

**Comment by sergio-sastre** [comment]
> Hey @sergio-sastre!
> Could you merge the latest master? We updated/fixed our examples.
> @AzamatCherchesov please review too and merge the PR if everything is good.

@AzamatCherchesov Done :)

**Review by AzamatCherchesov — APPROVED** [review]

**Review by Nikitae57 — APPROVED** [review]

**Pull request merged**